README.md

```diff
@@ -34,102 +32,104 @@ _/bricks/lean-welcome.html_:
 ```html
 <template>
   <h1>Welcome to LEGO</h1>
-  <p>The lean way to make web-components <slot></slot></p>
+  <p>The lean way to make ${state.duration} Web Components <slot></slot></p>
 </template>
+
+<script>
+  export default class extends Lego {
+    state = { duration: 'Short-Term' }
+  }
+</script>
 ```
 
-Later you can use it anywhere in your HTML:
+Later, when builded and loaded as a plain JS-file, you can use it anywhere in your HTML:
 
 ```html
-<lean-welcome>powerful and easy</lean-welcome>
+<lean-welcome duration="Future-Proof">powerful and easy</lean-welcome>
 ```
 
 Will read in your HTML:
 
+```html
+<h1>Welcome to LEGO</h1>
+<p>The lean way to make Future-Proof Web Components powerful and easy</p>
 ```
-Welcome to LEGO
-The lean way to make web-components powerful and easy
-```
-
-Of course there's **so much** more you can do.
-The [documentation](https://lego.js.org/) will guide you 📓
 
-_Pretty neat!_
+Of course there's **so much** more you can do. The [documentation](https://lego.js.org/) is here to guide you. 📓
 
+_Pretty neat!_ <!-- FIXME: What is it related to?  -->
```

Comment:
:warning: I don't really know what to do with this line. What is pretty neat in the previous lines?